src/renderer_vk.cpp

```diff
@@ -4774,7 +4768,7 @@ VK_DESTROY
 	}
 
 
-	void ScratchBufferVK::flush()
+	void ScratchBufferVK::flush_and_reset()
```

Comment:
Following bgfx code style would be `flushAndReset`, but it's better to implement as `reset(bool _flush = true)`

Reply:
Went with `flush(_reset = true)`, because that makes a lot more sense to me.